main-app/app/scripts/gameboard/gameboard-controller/gameBoard.js

```diff
@@ -0,0 +1,16 @@
+angular.module('Tombola.MyModule')
```

Comment:
Next cut: Rename this controller to what it does - then re-name the file it is in if different.